edit/location/location_test.go

```diff
@@ -16,7 +17,7 @@ var (
 		{"/home/dir", 100},
 		{"/foo/\nbar", 77},
 		{"/usr/elves/elvish", 6},
-	}, "/home")
+	}, "/home", eval.NewEvaler(), matchPattern)
```

Comment:
Please add a test using an alternative matcher. The alternative matcher can be a "toy" one, e.g. one that only matches directories starting with `/usr` regardless of the pattern.

Reply:
Done.